### ISSUE TITLE
Feature Request: Start Kanri with board last edited

### ISSUE THREAD

**Issue by PeterP2** [trobonox/kanri#317]
Or even better make it adjustable to open either on home page, a specific board, the last edited board

**Comment by trobonox** [comment]
Good idea. As soon as work on some more settings will be done, this could be an option for sure. Not 100% sure about the exact imlpementation but will figure it out as soon as I get to it.
